xml/image_studio_pointer.xml

```diff
@@ -20,7 +20,7 @@
     xlink:href="http://susestudio.com/"/>, there is also &suseonsite;. It is a
     &susestudio; release intended for installation on a server machine behind
     the firewall.</para>
-        <para>For the &suseonsite; documentation, see <link xlink:href="https://www.suse.com/documentation/suse_studio/"/>. For the
+        <para>For the &suseonsite; documentation, see <link xlink:href="https://documentation.suse.com/suse_studio/"/>. For the
```

Comment:
Please just reset this file. We don't have any SUSE Studio docs (because it hasn't been supported in 2 years) and we don't seem to publish this SBP document anymore either.

If you come across more SUSE Studio links (in other documentation that is still published), link to https://studioexpress.opensuse.org/.

Reply:
This file is in the repo, but just as a backup - why is it listed in the list of docs with links to be fixed? Only published papers should be on that list.